app/mailers/audit_objects_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# Send report of invalid objects to admin team
+class AuditObjectsMailer < ActionMailer::Base
+
+  layout 'mailer'
+
+  def error_report(bad_objects, stats)
+    @bad_objects = bad_objects
+    @stats = stats
+    @email = Avalon::Configuration.lookup('email.errors')
```

Comment:
Guessing this pulls from avalon.yml? Should the example file be updated to include this? https://github.com/ualbertalib/avalon/blob/master/config/avalon.yml.example?